dasharo/README.md

```diff
@@ -0,0 +1,66 @@
+# How to generate new Dasharo Release Signing Key?
+
+## Generate key with correct name
+
+For details about Dasharo keys naming please read [documentation]().
```

Comment:
so we still lack the documentation about naming?

Reply:
grr, missed that, let me try to fix right now

Reply:
@miczyg1 fixed